### PR TITLE
client_url is hardcoded in slave.pp

### PR DESCRIPTION
**Pull request by zonArt**
This PR intends to give the possibility to change it

### CODE REVIEW

File: manifests/slave.pp [hunk 1]
```diff
@@ -93,12 +93,13 @@
   $install_java             = $jenkins::params::install_java,
   $ensure                   = 'running',
   $enable                   = true
```
Comment (jhoblitt) [inline manifests/slave.pp]: missing comma

File: manifests/slave.pp [hunk 2]
```diff
@@ -93,12 +93,13 @@
...
+  $maven_repo_url           = 'http://maven.jenkins-ci.org/content/repositories/releases',
```
Comment (jhoblitt) [inline manifests/slave.pp]: It's probably not important that the download path is actually a maven repo.  Would it be better to have a param to specify the complete url to the jar?

Comment (zonArt) [inline manifests/slave.pp]: done

**Comment by jhoblitt** [comment]
@zonArt Could you squash these commits down and give us a headsup when you feel this PR is ready?

**Comment by rtyler** [comment]
@jhoblitt IMHO squashing commits isn't necessary, it's often nice but meh

**Pull request merged**